pandas/_libs/lib.pyx

```diff
@@ -1005,7 +1005,7 @@ _TYPE_MAP = {
     'complex64': 'complex',
     'complex128': 'complex',
     'c': 'complex',
-    'string': 'bytes',
+    'string': 'string',
     'S': 'bytes',
```

Comment:
Do you know if "S" is ever used anymore?

Reply:
You still get "S" with numpy strings (bytes):

```
In [18]: np.array(["a", "b"], dtype="S") 
Out[18]: array([b'a', b'b'], dtype='|S1')

In [19]: np.array(["a", "b"], dtype="S").dtype
Out[19]: dtype('S1')

In [20]: np.array(["a", "b"], dtype="S").dtype.name 
Out[20]: 'bytes8'

In [21]: np.array(["a", "b"], dtype="S").dtype.kind 
Out[21]: 'S'
```

I was more wondering what the original "string" -> "bytes" mapping could be for, but I can't think of a case where "string" is returned from a numpy dtype's name or kind

Reply:
SGTM.